services/api/src/helpers/billingGroups.ts

```diff
@@ -42,7 +29,7 @@ interface IGroup {
 
 export const getAllProjectsNotInBillingGroup = async () => {
   const keycloakAdminClient = await getKeycloakAdminClient();
-  const sqlClient = getSqlClient(USE_SINGLETON);
+  const sqlClient = getSqlClient();
```

Comment:
w/o the singleton mechanic, any use of `getSqlClient()` outside the apollo context must explicitly close its own connection. I'm not sure if that still applies with this one-off, non-long-running script though. I would assume that when the node process ends, everything gets cleaned up, but I've been wrong about that before, and we clean up the connections in the other cli scripts (eg https://github.com/amazeeio/lagoon/blob/master/services/api/src/helpers/sync-groups-opendistro-security.ts#L41)

So to be explicit here, you'd add `sqlClient.destroy()` somewhere before this function ends. Fair warning with this change, for some reason the act of closing the connection keeps the node process running for at least a minute after all the work has been completed.

Reply:
cool. will add this in. Not sure how often this helper will ever get run, but good to put this in.

Reply:
fixed in upcoming PR